static/scss/components/banners.scss

```diff
@@ -135,6 +80,13 @@
         display: none;
     }
 
+    .c-notification-banner-label {
+        color: $color-purple-50;
+
+        @include text-body-sm;
+        margin-bottom: $spacing-sm;
+    }
```

Comment:
Added this for the "action step" text. I think that could be applied to future/all banners.
 
<img width="319" alt="image" src="https://user-images.githubusercontent.com/2692333/138209377-6a5e4164-cf74-4bf0-add2-06abb487bdcb.png">